src/python/tests/test_update_container_definitions.py

```diff
@@ -33,7 +33,7 @@ def test_invalid_image_update_param_container(self):
         self.assertRaises(ValueError,
                           run, TestContainerDefinitionsUpdate.task_dfn_multi_containers,
                           'acontainer=web,container=timer,image-and-tag=golang', '')
-
+        
```

Comment:
Could you remove the extra whitespace as it causes a lint error?

Reply:
Will do